replica/distributed_loader.cc

```diff
@@ -38,6 +38,15 @@ static const std::unordered_set<std::string_view> system_keyspaces = {
                 db::system_keyspace::NAME, db::schema_tables::NAME
 };
 
+// Not super nice. Adding statefulness to the file. 
```

Comment:
Changelog: reverts should explain why they are reverting. Or they are eligible for a counter-revert.

Reply:
I put the main motivator in patch series log. Can copy into revert patch log as well.